followthemoney/types/common.py

```diff
@@ -51,21 +55,33 @@ class PropertyType(object):
     def docs(self) -> Optional[str]:
         return self.__doc__
 
-    def validate(self, text: Any, **kwargs) -> bool:
+    def validate(self, text: str) -> bool:
         """Returns a boolean to indicate if the given value is a valid instance of
         the type."""
-        cleaned = self.clean(text, **kwargs)
+        cleaned = self.clean(text)
         return cleaned is not None
 
-    def clean(self, text: Any, **kwargs) -> Optional[str]:
+    def clean(
+        self,
+        raw: Any,
+        fuzzy: bool = False,
+        format: Optional[str] = None,
+        proxy: Optional["EntityProxy"] = None,
+    ) -> Optional[str]:
         """Create a clean version of a value of the type, suitable for storage
         in an entity proxy."""
-        text = sanitize_text(text)
+        text = sanitize_text(raw)
         if text is None:
             return None
-        return self.clean_text(text, **kwargs)
+        return self.clean_text(text, fuzzy=fuzzy, format=format, proxy=proxy)
 
-    def clean_text(self, text: str, **kwargs) -> Optional[str]:
+    def clean_text(
+        self,
+        text: str,
```

Comment:
And here too, if feasible. Make the argument names consistent across the common definition and the overrides.